### PR TITLE
Stop passing err to logger.error

### PR DESCRIPTION
**Pull request by Beckyrose200**
https://github.com/DEFRA/water-abstraction-team/issues/45
https://github.com/DEFRA/water-abstraction-team/issues/42

Here we are trying to make the error logs more usable. Currently when an error happens the logs are getting jammed up with thousands of lines making them difficult to read and unusable. This change stops that pollution making them more usable.

### CODE REVIEW

File: src/lib/helpers/write-csv-to-s3.js [hunk 1]
```diff
@@ -41,12 +41,12 @@ module.exports = {
 
       return fs.unlink(filePath, err => {
         if (err) {
-          logger.error(new Error(err))
+          logger.error(new Error(err.stack))
```
Comment (Cruikshanks) [inline src/lib/helpers/write-csv-to-s3.js:44]: Not a thing you've done @Beckyrose200 . But curious as to why they took an error and then wrapped it in a `new Error()`.

I suspect we could simplify things

```suggestion
          logger.error(err.stack)
```

File: src/lib/helpers/write-csv-to-s3.js [hunk 2]
```diff
@@ -41,12 +41,12 @@ module.exports = {
...
         }
         logger.info(`File no longer relevant - deleting ${dataOut.Location} from the local server`)
       })
     } catch (e) {
-      logger.error(new Error(e))
+      logger.error(new Error(e.stack))
```
Comment (Cruikshanks) [inline src/lib/helpers/write-csv-to-s3.js:49]: Same again. Weird that it's doing this.

```suggestion
      logger.error(e.stack)
```

**Review by Cruikshanks — COMMENTED** [review]
Just a bit of weirdness in what they were doing with errors in a couple of places.

I'd suggest making the suggested changes and then confirming it still runs, and the tests still pass. If they do we might as well tidy these bits up.

File: src/lib/helpers/write-csv-to-s3.js (hunk 1 above)
Comment (Beckyrose200) [reply src/lib/helpers/write-csv-to-s3.js]: Made the above changes and the tests still pass :) (although there is only 30 of them)

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**